app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The use of class variable `people` to store instances of `Person` is not a good idea. It would be better to manage instances of `Person` outside of the class. This would make the `Person` class more reusable and easier to test.